Simplify the XML-JSON conversion BBE (#2965)
ballerina-platform/ballerina-distribution: examples/xml-from-json-conversion/xml_from_json_conversion.md

> # XML JSON conversion

Reviewer: is the title correct? this name should be consistent with XML-Record BBEs

> The `xmldata` library provides API to perform conversion from JSON to XML.

Reviewer 2: ```suggestion
The `xmldata` library provides an API to perform conversions from JSON to XML.
```

> For more information on the underlying module, see the [xmldata module](https://lib.ballerina.io/ballerina/xmldata/latest/).

Reviewer 2: ```suggestion
For more information on the underlying module, see the `[xmldata` module](https://lib.ballerina.io/ballerina/xmldata/latest/).
```

::: code xml_from_json_conversion.bal :::

> To run this sample use the `bal run` command.

Reviewer 2: ```suggestion
To run this sample, use the `bal run` command.
```
